mantidproject/mantid
# Change peak zoom level for SliceViewer

Fixes #19337 

Changes the zoom level on to peak. You should be able to see the entire peak 

**To test:**

1. Load a workspace with a corresponding spherical peak workspace
2. Open both in the slice viewer. Leave the default window size.
3. Select a peak. Confirm that the entire peak is visible

**Release notes**
See [here](https://github.com/mantidproject/mantid/pull/19338/commits/23ba03f5df3386f049eca4a0871014ee6ec85dd0)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
